### PR TITLE
docs: Update README

### PR DESCRIPTION
**Pull request by BenitoVisone**

### CODE REVIEW

File: .github/workflows/build-and-deploy.yml [hunk 1]
```diff
@@ -12,6 +12,9 @@ on:
     branches:
       - "main"
       - "release-*"
+    paths-ignore:
+      - "**.md"
+      - "**.yml"
```
Comment (rodoviti) [inline .github/workflows/build-and-deploy.yml:17]: why ignore only these two type of files and why all yml file in all subpaths?

i suggest to use an "allowed-list" strategy, so:
```suggestion
    paths:
      - "**/src/main/**"
      - "**pom.xml"
      - "Dockerfile"
      - ".github/workflows/build-and-deploy.yml"
```

File: .github/workflows/code-review.yml [hunk 2]
```diff
@@ -4,6 +4,10 @@ on:
     branches:
       - main
       - release-*
+    paths-ignore:
+      - "**.md"
+      - "**.yml"
```
Comment (rodoviti) [inline .github/workflows/code-review.yml:9]: why ignore only these two type of files and why all yml file in all subpaths?

i suggest to use an "allowed-list" strategy, so:
```suggestion
    paths:
      - "**/src/main/**"
      - "**pom.xml"
      - "Dockerfile"
      - ".github/workflows/code-review.yml"
```

**Review by rodoviti — CHANGES_REQUESTED** [review]

File: .github/workflows/build-and-deploy.yml (hunk 1 above)
Comment (BenitoVisone) [reply .github/workflows/build-and-deploy.yml]: fixed with https://github.com/pagopa/pdv-ms-tokenizer/pull/23/commits/acec6aa1b4329f4e47887412b40d02be086cad2c

File: README.md [hunk 3]
```diff
@@ -1 +1,94 @@
-# pdv-ms-tokenizer 
+# PDV MS-Tokenizer
+
+Spring Application implementing the **Tokenizer** microservice for the **Personal Data Vault** Project.
+
+This microservice works in conjunction with:
+
+- [pdv-ms-person](https://github.com/pagopa/pdv-ms-person)
+- [pdv-ms-user-registry](https://github.com/pagopa/pdv-ms-user-registry)
```
Comment (rodoviti) [inline README.md:8]: from the point of view of the _tokenizer_, it is not aware of the existence of other microservices in the same _ecosystem_. So, I suggest splitting dependencies into upstream and downstream
```suggestion
---

## Dependencies

Upstream:
- [pdv-ms-user-registry](https://github.com/pagopa/pdv-ms-user-registry)

Downstream:
- [Amazon DynamoDB](https://aws.amazon.com/dynamodb/?nc1=h_ls)
```

Comment (BenitoVisone) [reply README.md:8]: in these lines I wanted to highlight that this micro-service is part of the Personal Data Vault project and to give a global point of view I added these two links to easily navigate them. I think that we could keep both the information.

```suggestion
This microservice is part of the Personal Data Vault Project and works in conjunction with:

- [pdv-ms-person](https://github.com/pagopa/pdv-ms-person)
- [pdv-ms-user-registry](https://github.com/pagopa/pdv-ms-user-registry)

## Dependencies
Upstream:
- [pdv-ms-user-registry](https://github.com/pagopa/pdv-ms-user-registry)
Downstream:
- [Amazon DynamoDB](https://aws.amazon.com/dynamodb/?nc1=h_ls)

```

**Review by rodoviti — APPROVED** [review]

**Pull request merged**